content/en/metrics/faq/update-to-query-value-widget.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Update to Distribution Metrics Workflow FAQ
+kind: faq
+is_beta: true
+further_reading:
+- link: "/metrics/faq/update-to-query-value-widget"
+  tag: "FAQ"
+  text: "Update to Top Lists and Query Value Widgets"
```

Comment:
```suggestion
```
This would add a link at the bottom of the page to this page itself. I don't think that's what you want.